src/app/projects/states/portfolio/directives/portfolio-welcome-step/portfolio-welcome-step.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input, Inject, OnInit } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { DoubtfireConstants } from 'src/app/config/constants/doubtfire-constants';
+
+@Component({
+  selector: 'portfolio-welcome-step',
+  templateUrl: 'portfolio-welcome-step.component.html',
+  styleUrls: ['portfolio-welcome-step.component.scss']
+})
+export class PortfolioWelcomeStepComponent {
+  @Input() advanceActiveTabs: any;
+  constructor(private constants: DoubtfireConstants) { }
+  public externalName: BehaviorSubject<string> = new BehaviorSubject<string>('OnTrack');
+
+  public advanceActiveTab(input: any) {
+    return this.advanceActiveTabs(input)
+  }
+
+  ngOnInit(): void {
+    this.externalName = this.constants?.ExternalName;
```

Comment:
The externalName is an Observable, even we can get value using method ` getValue()` or ` value`. The better way to do it would be the subscribe. Check this article,[The only way you should be getting values "out of" an Observable/Subject is with subscribe!](https://stackoverflow.com/a/45227115)